items/index.js

```diff
@@ -0,0 +1,80 @@
+const ItemTypes = require('./ItemTypes');
+
+const enhancementMap = {
+  '16': 'PRI',
+  '17': 'DUO',
+  '18': 'TRI',
+  '19': 'TET',
+  '20': 'PEN',
```

Comment:
Numbers that are keys don't have to be strings! For example:
```javascript
const enhancementMap = {
  16: 'PRI',
  17: 'DUO',
  18: 'TRI',
  19: 'TET',
  20: 'PEN'
};
```

Reply:
Also, say you planned on adding more of these later, and in case you change where they start, you could make them more programmatic:
```javascript
const enhancementMap = [
  'PRI', 'DUO', 'TRI', 'TET', 'PEN'
].map((e, i) => ({ [i+16]: e }));
```